### PR TITLE
Add signup screen

### PR DESCRIPTION
**Pull request by me-jterhorst**
Created Sign Up Page Layout,

### CODE REVIEW

File: client/src/pages/SignUp.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          <Link className="opaque link" to="/login">
+            Login instead
+          </Link>
+        </p>
+
+        <form className="dispFlex margin-b--l">
+          <InputField id="Signup" label="Username" />
+          <InputField id="Signup" label="Email" />
+          <InputField id="Signup" label="Password" type="password" />
+          <InputField id="Signup" label="Repeat password" type="password" />
```
Comment (johnxgaehner) [inline client/src/pages/SignUp.js:25]: Hey Jakob, I think IDs should be unique and only used once. also in connection with the input a bit confusing. you use it for styling here, right?

File: client/src/pages/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import "./SignUp.css";
+import Card from "../components/Card";
+import BackButton from "../components/BackButton";
+import FooterSubmit from "../components/Card_components/FooterSubmit";
+import InputField from "../components/InputField";
+import { Link } from "react-router-dom";
+
+export default function SignUp() {
+  return (
+    <main id="Signup" className="card-screen">
```
Comment (johnxgaehner) [inline client/src/pages/SignUp.js:10]: here's another id.

**Review by johnxgaehner — COMMENTED** [review]
Hey Jakob, looks good!
you are using id more than once, this should be a class then, I think.

File: client/src/pages/SignUp.js (hunk 1 above)
Comment (me-jterhorst) [reply client/src/pages/SignUp.js:25]: Hey I can see where it get's confusing, it's actually not an id for an Element it's a prop that I am using within my Input to make the labels work in accordance with the input fields. 

I'll rename the property to address that

Comment (johnxgaehner) [reply client/src/pages/SignUp.js]: ah, got that! sorry. makes sense now!

**Review by patrikglanemann — COMMENTED** [review]
The signup page layout looks good to me so far:)

File: client/src/pages/SignUp.js (hunk 1 above)
Comment (me-jterhorst) [reply client/src/pages/SignUp.js]: No it's definitely a valid concern :) Thank you for your input

**Pull request merged**